Cargo.toml

```diff
@@ -53,15 +53,18 @@ debug = []
 integration_test = []
 
 [dependencies]
+unicode-segmentation = { version = "1" }
+unicode-width = { version = "0.2" }
+
+# Optional dependencies
```

Comment:
Also please let the options ordered :)

They're auto-ordered via taplot.

Reply:
ansi-str < console in lex order so it should be ordered already?